Tests/Shared/CustomAssertions/AssertAsync.swift

```diff
@@ -336,7 +336,7 @@ struct AssertAsync {
     }
 }
 
-@_functionBuilder
+@resultBuilder
```

Comment:
This will probably cause issues for projects building on Xcode versions below 13.0

Reply:
Hmmm, that could be true. Reverting